parm/config/config.resources

```diff
@@ -159,7 +124,7 @@ elif [ $step = "eobs" -o $step = "eomg" ]; then
     export wtime_eobs="00:30:00"
     export wtime_eomg="01:00:00"
     if [ $CASE = "C768" ]; then
-      export npe_eobs=84
+      export npe_eobs=400
     elif [ $CASE = "C384" ]; then
       export npe_eobs=42
```

Comment:
Do the lower resolutions need a similar increase for npe_eobs? Going from 400 for C768 to 42 for C384 seems too big of a drop now.

Reply:
Need to test and make changes in the future.